src/internal/Subject.ts

```diff
@@ -102,21 +97,19 @@ export class Subject<T> extends Observable<T> implements SubscriptionLike {
 
   /** @internal */
   protected _trySubscribe(subscriber: Subscriber<T>): TeardownLogic {
```

Comment:
This method can probably be removed, since it just propagades the call to parent class.